src/Sparrow/Json/JsonDeserializationNoIgnoreAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Sparrow.Json
+{
+    /// <summary>
+    /// Instructs the <see cref="JsonDeserializationBase"/> to serialize nonPublic field .
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property)]
+    internal sealed class JsonDeserializationNoIgnoreAttribute : Attribute
```

Comment:
`NoIgnore` is an awkward name, can we find something better?

Reply:
`ForceJsonSerializationAttribute` ?

Reply:
ok. done